style.css

```diff
@@ -0,0 +1,355 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
since you did this imports here, is it still necessary to have that google fonts script in your html's head?